### ISSUE TITLE
The Recording layer turns on by itself

### ISSUE THREAD

**Issue by raindropsfromsky** [osmandapp/Osmand#3035]
I am an avid OSM mapper, and whenever I am in new parts of the country, I always take photos and audio notes using OsmAnd, and later edit OSM using iD.

As a result, my map always shows a lot of pointers for photos and audio notes.

To keep the map free of clutter, I always keep the Recording Layer turned off. 
(I have done that in both **Browse** and **Car** modes)

But somehow the Recording Layer turns on automatically after some time, when I unlock the device, or when I restart the phone. 

**Desired:** This setting should be persistent.

**Comment by sonora** [comment]
@vshcherb : I guess we automatically turn on the recording layer after each recording?https://github.com/osmandapp/Osmand/blob/master/OsmAnd/src/net/osmand/plus/audionotes/AudioVideoNotesPlugin.java#L1533

@raindropsfromsky : To reduce complexity the setting is global, not profile-specific, so it does not matter in which profile you turn it on or off.

**Comment by raindropsfromsky** [comment]
No, I have observed that even when I have NOT added any note (of any type), the Recording layer gets turns on.

**Comment by daganzdaanda** [comment]
Hi, I've noticed the same or similar behaviour for a while now, too. 
Whenever I start recording a GPS track, the layer with all the photo/audio notes gets activated (and clutters the map). I can turn it off, but that setting is never remembered. 
The behaviour used to be different, though I can't recall when it changed. I think that the "recorded notes" layer only turned on when a new note was added, or when I explicitly turned it on. That makes more sense IMHO.
